draft-ietf-tsvwg-rfc4960-bis.xml

```diff
@@ -217,7 +217,7 @@ addresses.</t>
 </dd>
 <dt>Bundling:</dt>
 <dd>
-<t>An optional multiplexing operation, whereby more than one user message can
+<t>An optional multiplexing operation, whereby more than one user or control message can
```

Comment:
We are referring to the bundling of user messages. I don't think we use the term `control message` in the document nor is combining control information and user data in a packet uncommon.
I changed `Bundling:` to `Bundling of User Messages:`

Reply:
Good point, we just have "control chunk"s but not "control message"s; I missed that.

Reply:
Just to be crystal clear: I only changed `Bundling:` to `Bundling of User Messages:` and kept the text. I think the intention was not to define bundling of chunks (where your text would apply) but bundling of user messages (where bundling of control chunks is not relevant).

Reply:
OK. I added:
```
<dt>Bundling Of Chunks:</dt>
<dd>
<t>An optional multiplexing operation, whereby more than one chunk can
be carried in the same SCTP packet.</t>
</dd>
```
I'm marking this as resolved. If you don't agree, please unresolve the conversation.

Reply:
For the record, my :+1: was intended to signal that your change was fine.  I am quite ambivalent about whether there is a "Bundling of Chunks" section -- do whatever is easiest for you.  Sorry about the confusing signal.

Reply:
I think it can't hurt to have it. So I prefer to keep it. At least it now makes crystal clear that bundling of chunks on the sender side is optional...